raiden/tests/unit/transfer/mediated_transfer/test_mediation_fee.py

```diff
@@ -220,3 +223,46 @@ def test_get_lock_amount_after_fees(flat_fee, prop_fee, initial_amount, expected
         lock=lock, payer_channel=payer_channel, payee_channel=payee_channel
     )
     assert locked_after_fees == expected_amount
+
+
+@pytest.mark.parametrize(
+    "flat_fee, prop_fee, imbalance_fee, initial_amount, expected_amount",
+    [(0, 0, 10_000, 50_000, 50_000 + 1996)],
```

Comment:
I get 2000 instead of 1996 in my sheet. I'll need to find out where the difference comes from.

Reply:
The 2000 was for the new IP function. It's 1998 for the old one. I'll chalk the difference up to different rounding behaviour and do more thorough checking when the new function is in.